onedal/datatypes/numpy_helpers.hpp

```diff
@@ -81,18 +81,24 @@
             _FUNCT(double);                     \
             break;                              \
         }                                       \
+        case NPY_INTLTR:                        \
         case NPY_INT32: {                       \
             _FUNCT(std::int32_t);               \
             break;                              \
         }                                       \
+        case NPY_UINTLTR:                       \
         case NPY_UINT32: {                      \
             _FUNCT(std::uint32_t);              \
             break;                              \
         }                                       \
+        case NPY_LONGLTR:                       \
+        case NPY_LONGLONGLTR:                   \
         case NPY_INT64: {                       \
             _FUNCT(std::int64_t);               \
             break;                              \
         }                                       \
+        case NPY_ULONGLTR:                      \
+        case NPY_ULONGLONGLTR:                  \
```

Comment:
Why are you updating this helper in current PR? this should be separate branch work

Reply:
We need this fix to convert numpy array of np.int32 to table in this algorithm

Reply:
Could you link the lines of code which utilizes the int table conversion?

Reply:
> this fix to convert numpy array of np.int32 to table in this algorithm

the is no problem for the fix. Why it is in the current PR? This kind of changes should be separated

Reply:
There was a problem during the build if these lines were not added

Reply:
You can see that all double and float types already had case NPY_DOUBLELTR: